common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
```

Comment:
@explorerleslie could you now take a look at this? http://studio.hinting.m.sandbox.edx.org/course/OLIx/Hint101x/HintingTest1 - you'll be able to see the changes made to these yaml files by adding a new problem component (any type) and seeing the revised wording.